packages/next-bundle-analyzer/index.d.ts

```diff
@@ -4,6 +4,12 @@ declare function NextBundleAnalyzer(options?: {
   enabled?: boolean
   openAnalyzer?: boolean
   analyzerMode?: 'json' | 'static'
+
+  /**
+   * Log level. Can be "info", "warn", "error" or "silent".
+   * @default 'info'
+   */
+  logLevel?: "info" | "warn" | "error" | "silent" | undefined
```

Comment:
Let's align the quotes
```suggestion
  /**
   * Log level. Can be 'info', 'warn', 'error' or 'silent'.
   * @default 'info'
   */
  logLevel?: 'info' | 'warn' | 'error' | 'silent' | undefined
```

Reply:
Thank you! I've merged your change. I copy-pasted this from the bundleAnalyzer repo, so I didn't notice the style change until after I commit. 😔